collections/_posts/2023-07-06-Mean_teachers_are_better_role_models.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: review
+title: Mean teachers are better role models: Weight-averaged consistency targets improve semi-supervised deep learning results
+tags: machine-learning, neural and evolutionary computing
+author: Tarvainen Antti, Valpola Harri
+cite:
+    authors: "Tarvainen et Valpola"
+    title:   "Mean teachers are better role models: Weight-averaged consistency targets improve semi-supervised deep learning results"
+    venue:   "Neural Information Processing Systems 30 (NIPS 2017) pre-proceedings"
+pdf: https://arxiv.org/pdf/1703.01780.pdf
+---
+
+# Highlights
+
+- Method : The mean teachers model **averages model weigths**, instead of label predictions like the Temporal Ensembling method. Allows for more accurate targets and faster feedback between the student and teacher models.
```

Comment:
Here, since you're talking about the method directly, I would use the singular for teachers:
```suggestion
- Method : The mean teacher model **averages model weigths**, instead of label predictions like the Temporal Ensembling method. Allows for more accurate targets and faster feedback between the student and teacher models.
```